src/components/TokenAmount/TokenAmount.js

```diff
@@ -47,6 +47,7 @@ const TokenAmount = React.memo(function TokenAmount({
               padding-right: ${size === 'large' ? 0.5 * GU : 0.25 * GU}px;
               ${textStyle(size === 'large' ? 'title2' : 'body2')};
               line-height: 1;
+              ${size === 'large' ? `margin-bottom: -2px;` : ''}
```

Comment:
Just to be safe, we could use `transform: translateY(-2px)` so that it can’t impact any surrounding element:

```suggestion
              ${size === 'large' ? `transform: translateY(2px);` : ''}
```